srcpkgs/nvidia/template

```diff
@@ -1,41 +1,40 @@
 # Template file for 'nvidia'
 
-_desc="NVIDIA drivers for linux (long-lived series)"
+_desc="NVIDIA drivers for linux (short-lived series)"
```

Comment:
I would just remove the last part. This way it could look like the driver was supposed to stay on this series now which it doesn't have to.

Reply:
Removed.